modules/@apostrophecms/piece-type/ui/apos/components/AposImageCropper.vue

```diff
@@ -1,11 +1,23 @@
 <template>
-  <div class="apos-image-cropper">
+  <div
+    class="apos-image-cropper"
+    @click="onImageClick"
+  >
+    <span
+      class="apos-image-focal-point"
+      ref="focalPoint"
+      v-apos-tooltip="'apostrophe:focalPoint'"
+      @mousedown="onFocalPointMouseDown"
+    />
     <cropper
       ref="cropper"
       :src="attachment._urls.uncropped
         ? attachment._urls.uncropped.original
         : attachment._urls.original"
-      @change="onChange"
+      :stencil-props="{ 'data-stencil': '' }"
```

Comment:
_ℹ️  This is the only way I managed to get a hold of the stencil rectangle element_